modules/ztp-generating-install-and-config-crs-manually.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-generating-install-and-config-crs-manually_{context}"]
+= Generating ZTP installation and configuration CRs manually
+
+Use the `generator` entrypoint for the `ztp-site-generate` container to generate the site installation and configuration custom resource (CRs) for a cluster based on `SiteConfig` and `PolicyGenTemplate` CRs.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create an output folder by running the following command:
++
+[source,terminal]
+----
+$ mkdir -p ./out
+----
+
+. Export the `argocd` directory from the `ztp-site-generate` container image:
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run --log-driver=none --rm registry.redhat.io/openshift4/ztp-site-generate-rhel8:v{product-version} extract /home/ztp --tar | tar x -C ./out
```

Comment:
What is the product-version mapped to?
Note that starting from 4.11, the image tag is changed to match the zstream version. 
e.g., latest 4.11 ztp site generate image is this:

`registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.11.1`

Reply:
{product-version} gets incremented each release as follows: 4.10, 4.11, 4.12, etc.

Reply:
So I will update above as `podman run --log-driver=none --rm registry.redhat.io/openshift4/ztp-site-generate-rhel8:v{product-version}.1`